kubernetes/helm/README.md

```diff
@@ -0,0 +1,74 @@
+# Artifactory in Kubernetes using Helm package manager
+This directory has examples for Artifactory in Kubernetes deployments using the [Helm](https://helm.sh/) package manager.
+
+## Helm
+From https://helm.sh/:
+```
+What is Helm?
+
+Helm helps you manage Kubernetes applications — Helm Charts helps you define, install, and upgrade even the most complex Kubernetes application.
+```
+
+Helm in [GitHub](https://github.com/kubernetes/helm)
+
+## Deployment
+
+### Installing helm
+Follow [instructions on installing helm](https://github.com/kubernetes/helm#install)
+ 
+#### Setup Helm to use your cluster
+To initialize Helm and install Tiller, you should run
+```bash
+$ helm init
+```
+
+### Deploying Artifactory Pro
+You can deploy the Artifactory Pro chart, which is in the [artifactory-pro](artifactory-pro) directory
+```bash
+$ helm install -n artifactory-pro ./artifactory-pro
+```
+
+You can package the Artifactory Pro chart and distribute it for use later
```

Comment:
- Packaging creates clutter that shouldn't be in commits. Please add packaged files to .gitignore.
- Consider adding a link to the sister open source project https://github.com/JFrogDev/artifactory-user-plugins/tree/master/helm/helmRepoSupport, and then add a step showing users how to push this chart to their private repo.
- Move to end of Readme to keep flow consistent. Alternatively, provide a link to the Helm FAQ if users are interested in packaging.